s3/session.go

```diff
@@ -86,8 +87,12 @@ func getDefaultConfig(settings map[string]string) *aws.Config {
 	// most services. If you want to implement custom retry logic, you can implement the
 	// request.Retryer interface.
 	config := defaults.Get().Config.
-		WithRegion(settings[RegionSetting])
-	config = request.WithRetryer(config, NewThrottlerRetries(MaxRetries))
+		WithRegion(settings[RegionSetting]).
+		WithLogger(aws.LoggerFunc(func(args ...interface{}) {tracelog.DebugLogger.Println(args...)})).
```

Comment:
Split line. We don't write expressions on the same line with a function signature.